docs/sources/enterprise/license-restrictions.md

```diff
@@ -23,9 +27,13 @@ In the context of licensing, each user is classified as either a viewer or an ed
 
 Restrictions are applied separately for viewers and editors.
 
-When the number of maximum active viewers or editors is reached, Grafana displays a warning banner.
+When the number of maximum active viewers or editors is reached, only them can log in. New users or non-active users can't log in.
+
+### Concurrent sessions limit
+
+Sometimes it is useful to log in to an account from multiple locations at the same time. With Grafana Enterprise 7.5+, accounts are limited to the number of concurrent sessions authorized in your license (default to three). A new session is created when a user logs in to Grafana from a new device, a different browser, or an incognito window, but not when a user logs in to Grafana from another tab or window within the same browser. Grafana can be open in many browser tabs while still using only one session.
```

Comment:
```suggestion
Sometimes it is useful to sign in to an account from multiple locations simultaneously. As of Grafana Enterprise 7.5.0, accounts are limited to the number of concurrent sessions authorized in your license, which is typically three. A new session is created when a user signs in to Grafana from a new device, a different browser, or an incognito window. If a user signs in to Grafana from another tab or window within the same browser, then only one licensed session is used.
```

Reply:
Do we want to say "you" instead of "a user" here or are we writing to an administrator in this context? My gut says, "If you sign in to ..." instead of "If a user signs in to ..."

Reply:
I applied your feedback, I just replaced the `licensed session` at the end with `session`. For the "you" vs "a user", I agree "you" sounds a bit better but I think it will be mostly administrators that will read this doc page, so I don't know. @mjseaman any opinion on this?

Reply:
I'm happy with "you" if that's our style. This does apply to _all_ users - as long as that comes through in the docs, we're good 👍